app/src/main/res/layout/activity_bill.xml

```diff
@@ -0,0 +1,287 @@
+
+        <LinearLayout
+            android:layout_width="match_parent"
+            android:layout_height="match_parent"
+            android:background="@color/color_grey_100"
+            android:orientation="vertical"
+            tools:context="com.framgia.fsalon.screen.bill.BillActivity"
+            >
+
+            <android.support.v7.widget.CardView
+                android:layout_width="match_parent"
+                android:layout_height="wrap_content"
+                android:layout_marginBottom="@dimen/dp_5"
+                bind:cardBackgroundColor="@color/color_white"
+                car_view:cardElevation="@dimen/dp1"
+                >
+
+                <LinearLayout
+                    android:layout_width="match_parent"
+                    android:layout_height="wrap_content"
+                    android:orientation="horizontal">
+
+                    <android.support.design.widget.TextInputLayout
+                        android:layout_width="0dp"
+                        android:layout_height="wrap_content"
+                        android:layout_gravity="center_vertical"
+                        android:layout_margin="@dimen/dp_10"
+                        android:layout_weight="1">
+
+                        <EditText
+                            android:layout_width="match_parent"
+                            android:layout_height="wrap_content"
+                            android:hint="@string/title_phone"
+                            android:inputType="textPersonName"
+                            android:textSize="@dimen/sp_12"/>
+                    </android.support.design.widget.TextInputLayout>
+
+                    <ImageView
+                        tools:ignore="ContentDescription"
+                        android:layout_width="wrap_content"
+                        android:layout_height="wrap_content"
+                        android:layout_gravity="center_vertical"
+                        android:paddingEnd="@dimen/dp_10"
+                        app:srcCompat="@drawable/ic_search_40dp"/>
+                </LinearLayout>
+            </android.support.v7.widget.CardView>
+
+            <android.support.v7.widget.CardView
+                android:layout_width="match_parent"
+                android:layout_height="wrap_content"
+                android:layout_marginBottom="@dimen/dp_5"
+                android:padding="@dimen/dp_5"
+                bind:cardBackgroundColor="@color/color_white"
+                car_view:cardElevation="@dimen/dp1"
+                >
+
+                <LinearLayout
+                    android:layout_width="match_parent"
+                    android:layout_height="wrap_content"
+                    android:orientation="vertical">
+
+                    <LinearLayout
+                        android:layout_width="match_parent"
+                        android:layout_height="wrap_content"
+                        android:layout_marginStart="@dimen/dp_10">
```

Comment:
ERROR: [RtlCompat] To support older versions than API 17 (project specifies 15) you should *also* add `android:layout_marginLeft="@dimen/dp_10"`
 API 17 adds a `textAlignment` attribute to specify text alignment. However, if you are supporting older versions than API 17, you must *also* specify a gravity or layout_gravity attribute, since older platforms will ignore the `textAlignment` attribute.